### DISCUSSION TITLE
Any examples of server.read for version 2.0.2?

### DISCUSSION THREAD

**Discussion by theotherjenkutler** [savonet/liquidsoap#2240] [Q&A]
Hello again and congratulations on releasing the next update!

I'm trying to figure out how to use server.read. I couldn't find any examples for version 2.0.2. Is this not in use anymore? Is there anything you can recommend to use for something similar? Thank you thank you

**Comment by toots** [comment]
Thanks!

Sadly, it's been removed. We hadn't seen any use of it in years and it was adding a lot of complexity to the core.. 😔
